tests/testsuite/ssh.rs

```diff
@@ -184,10 +184,9 @@ fn known_host_works() {
     // Validate the fingerprint while we're here.
     let fingerprint = stderr
         .lines()
-        .find(|line| line.starts_with("  The ECDSA key fingerprint"))
+        .find_map(|line| line.strip_prefix("  The ECDSA key fingerprint"))
```

Comment:
This isn't quite equivalent, and is causing a test failure. I think it needs to be something like:

```suggestion
        .find_map(|line| line.strip_prefix("  The ECDSA key fingerprint is: "))
```